tljh/installer.py

```diff
@@ -208,6 +208,7 @@ def ensure_jupyterhub_package(prefix):
         'jupyterhub-firstuseauthenticator==0.12',
         'jupyterhub-nativeauthenticator==0.0.4',
         'jupyterhub-ldapauthenticator==1.2.2',
+        'jupyterhub-tmpauthenticator==0.6',
```

Comment:
From what I understand, when a user wants to custom its juputerhub_config using *tljh_custom_jupyterhub_config* in order to use something that's not directly installed by TLJH, they could would use *tljh_extra_hub_pip_packages* to install it first. 

If this is the case, I believe tmpauthenticator package should/could come from *tljh_extra_hub_pip_packages*.

Reply:
I've split this part of the PR to https://github.com/jupyterhub/the-littlest-jupyterhub/pull/365 separately, and based this PR on that. So we have two PRs:

1. #365 that adds tmpauthenticator
2. This PR, which adds two hooks with tests for both of them. There is a test for `tljh_extra_hub_pip_packages` that imports the 'there' package, and the test for `tljh_custom_jupyterhub_config` that sets tmpauthenticator to be the default authenticator.

How does that sound?